protocol-designer/src/components/organisms/SelectLabwareModal/index.tsx

```diff
@@ -113,21 +100,15 @@ export function SelectLabwareModal(
   const customLabwareDefs = useSelector(getCustomLabwareDefsByURI)
   const has96Channel = getHas96Channel(pipetteEntities)
   const defs = getOnlyLatestDefs()
-  const lidLoadNames = Object.values(defs)
-    .filter(
-      def =>
-        def.allowedRoles?.includes('lid') &&
-        def.parameters.loadName !== TIPRACK_LID_LOADNAME
-    )
-    ?.map(def => def.parameters.loadName)
-
+  const universalLid = Object.entries(defs).find(
+    ([id, def]) => def.parameters.loadName === UNIVERSAL_LID_LOADNAME
+  )
```

Comment:
if we don't need the ID:

```suggestion
  const universalLid = Object.values(defs).find(
    ({ parameters }) => parameters.loadName === UNIVERSAL_LID_LOADNAME
  )
```